data/data-pipeline/data_pipeline/content/config/score_fields.yml

```diff
@@ -0,0 +1,56 @@
+# This is a temporary file. We should make sure this *type* of information is maintained when we refactor. 
+fields:
+    - column_name: Total threshold criteria exceeded
+      notes: Lists out the total number of criteria (where each category has one or more criteria) exceeded. For example, a tract that exceeds the 90th percentile for linguistic isolation (1) and unemployment (2), and meets the training and workforce development socioeconomic criteria (high school attainment rate and low percentage of higher ed students) would have a 2 in this field. 
+    - column_name: Definition M (communities)
```

Comment:
Should this be `Identified as disadvantaged (v0.1)` or whatever?

Reply:
It's the native field, so the one in usa.csv, not the downloaded field.

Reply:
oh i see